src/main/java/ch/njol/skript/expressions/ExprCommandBlockCommand.java

```diff
@@ -0,0 +1,83 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.block.Block;
+import org.bukkit.block.CommandBlock;
+import org.bukkit.entity.minecart.CommandMinecart;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Command Block Command")
+@Description(
+	"Gets or sets the command associated with a command block or minecart with command block."
+)
+@Examples({
+	"send command of {_block}",
+	"set command of {_cmdMinecart} to \"say asdf\""
+})
+@Since("INSERT VERSION")
+public class ExprCommandBlockCommand extends SimplePropertyExpression<Object, String> {
+
+	static {
+		register(ExprCommandBlockCommand.class, String.class, "[command[ ]block] command", "blocks/entities");
+	}
+
+	@Override
+	public @Nullable String convert(Object holder) {
+		String command = "";
+		if (holder instanceof Block && ((Block) holder).getState() instanceof CommandBlock) {
```

Comment:
feel free to use a pattern variable if you want, since this will be java 17

Reply:
I'll leave it as is for now so tests continue to pass, but I'll change it whenever the java 17 pr is merged